notebooks/ingest.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dotenv import dotenv_values\n",
+    "from pathlib import Path\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import nccid_cleaning.etl as etl\n",
+    "from nccid_cleaning import clean_data_df, patient_df_pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook can be used to generate CSV files containing patient clinical data, and image metadata for each patient and image file within the NCCID data. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use these tools you need to provide a `BASE_PATH` that points to the location of the data that has been pulled from the NCCID S3 bucket, where your local directory structure should match the original S3 structure. If you have split the data into training/test/validation sets, each subdirectory should have the same structure as the original S3 bucket and the below pipeline should be run separately for each of the dataset splits. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can either set the base path using a `.env` file (see <a href=\"https://github.com/theskumar/python-dotenv\">python-dotenv</a> and project README) or specify directly as string e.g., `\"/project/data/training/\"`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# replace base_path_str below\n",
+    "base_path_str = dotenv_values(\"../.env\")[\"training_data\"]\n",
```

Comment:
This is a neat use of dotenv. Though if the values are just parsed like this (and it's not loaded through the environment but .env is used as a configuration file), I wonder if it was simpler to just use the built in `configparser` library? https://docs.python.org/3.6/library/configparser.html

Reply:
changed to configparser!